cypress.json

```diff
@@ -5,5 +5,11 @@
   "supportFile": "cypress/support/index.js",
   "video": false,
   "defaultCommandTimeout": 10000,
-  "chromeWebSecurity": false
+  "chromeWebSecurity": false,
+  "pageLoadTimeout": 10000,
+  "blockHosts": ["vitals.vercel-insights.com", "*sentry.io", "*googletagmanager.com"],
+  "ignoreTestFiles": ["info.test.ts"],
```

Comment:
skip info test. e2e becomes fails alarm we literally ignore it and may miss some major error  🤷‍♂️

Reply:
Could we add a comment on this line so other devs could know why we ignore it? Thanks

Reply:
it's json. That's why I comment on this. I'll remove this once we know info is back to normal